### ISSUE TITLE
Inline text color & background Theme.json

### ISSUE THREAD

**Issue by tatof** [WordPress/gutenberg#39379]
## What problem does this address?
I would like to recreate the old function of inline color changes. (highlight)
I am building a theme with titles that uses two colors. This is at the moment impossible because the inline text color option disappeared in wordpress gutenberg.

This is what I want to create:
![image](https://user-images.githubusercontent.com/617637/157851940-4f2495c3-9350-431f-9c8e-0d04be0c45ae.png)

With an option like this:
![image](https://user-images.githubusercontent.com/617637/157851978-603c0414-efcb-4fbf-99ca-658dc387c9bd.png)

## What is your proposed solution?
An extra option for color in the theme.json called highligh-color & highligh-background. Or something like that.

Maybe there is a solution to this problem but I can't find anything online :(

**Comment by carolinan** [comment]
Hi!
In the image, it looks like your are using a third party block? (It is up to the developer of that custom block to add support for the functionality)

For text based WordPress and Gutenberg blocks, the inline text color option is in the block tools menu.
Please click on the arrow icon, then select highlight:
![animation showing how to select text highlight](https://user-images.githubusercontent.com/7422055/157883033-e0efccdd-c16a-48c3-bee1-98c29dc4dc8a.gif)

**Comment by tatof** [comment]
Hey,

Yes it was an example could not find the real version of gutenberg. (not using a third party block, just searched google for images)
Awesome gif! thats exactly what I need :) only i dont have it

How did you create that? how can I enable "highlight"?
![sdfsdf](https://user-images.githubusercontent.com/8288576/157890629-d5ca974b-ab6b-4a21-8cee-568c2a7deb51.png)

I tried to turn it on in the theme.json like this but doesnt work:

`{
    "version": 3,
    "settings": {
        "color": {
            "highlight": true,
        }
    }
}`

Its my own custom made theme and Ive searched if I disabled it in the past but can't find anything that breaks this.
Will switch to 2022 to see if it works there.

Edit: 2021 also no hightlight function. (dont have 2022 right now)
